client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/TestResultsDto.java

```diff
@@ -14,4 +16,9 @@ public class TestResultsDto {
     public List<AdditionalInfoDto> additionalInfoList = new ArrayList<>();
 
     public List<ExtraHeuristicsDto> extraHeuristics = new ArrayList<>();
+
+    /**
+     * Incremental DynaMOSA control-dependence graphs discovered since the last handshake.
+     */
+    public List<ControlDependenceGraphDto> dynamosaCdgs = new ArrayList<>();
```

Comment:
no ref to Dynamosa here

Reply:
`controlDependenceGraph`. and yes, there should be no reference to dynamosa under `client-java`

Reply:
acrnoyms like `cfg` are fine inside short-lived code blocks, but not for class fields